modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/GmosSouthController.scala

```diff
@@ -85,7 +85,7 @@ object GmosSouthController {
         case (BuiltinROI.CUSTOM, r) if r.nonEmpty => \/.right(new RegionsOfInterest(\/.right(r)) {})
         case _ => \/.left(Unexpected("Inconsistent values for GMOS regions of interest"))
       }
-    def unapply(r: RegionsOfInterest): Option[BuiltinROI \/ List[ROI]] = Some(r.rois)
+    def unapply(r: RegionsOfInterest): Some[BuiltinROI \/ List[ROI]] = Some(r.rois)
```

Comment:
It was an experiment actually to see if the pattern matching worked. I should probably revert it to `Option`